### PR TITLE
Define files that should not be exported to releases

### PR DESCRIPTION
**Pull request by fabiocarneiro**
This way we remove the not necessary files from the final release package(.zip) and reduce the download size when the library is download from dist with composer.

### CODE REVIEW

File: .gitattributes.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/benchmark export-ignore
+/bin export-ignore
+/vendor export-ignore
```
Comment (dbellettini) [inline .gitattributes.yml]: This is already in .gitignore, why do we need it here?

Comment (fabiocarneiro) [inline .gitattributes.yml]: gitignore is for files that should not go to commits, this is for release control. The purpose is different. Even if tomorrow you decide to add this folder to git, you might still want to exclude from your release.

File: .gitattributes.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/.php_cs export-ignore
+/.scrutinizer.yml export-ignore
+/phpunit.xml.dist export-ignore
```
Comment (dbellettini) [inline .gitattributes.yml]: not sure about this last one, why do we keep tests and skip test configuration?

Comment (fabiocarneiro) [inline .gitattributes.yml]: You're right. I missed the test folder lol.

**Pull request merged**